contracts/contracts/RequestManager.sol

```diff
@@ -475,12 +480,14 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
         }
 
         if (msg.sender == claimer) {
-            claimerStake += msg.value;
+            claimerStake += uint96(msg.value);
             claim.claimerStake = claimerStake;
         } else {
-            claim.lastChallenger = msg.sender;
-            claim.challengersStakes[msg.sender] += msg.value;
-            challengerStakeTotal += msg.value;
+            if (msg.sender != lastChallenger) {
+                claim.lastChallenger = msg.sender;
+            }
```

Comment:
We should remove this `if` and update the `lastChallenger` unconditionally, as before.